Source/WebCore/css/CSSRule.idl

```diff
@@ -49,4 +49,5 @@
     [ImplementedAs=CounterStyle, EnabledBySetting=CSSCounterStyleAtRulesEnabled] const unsigned short COUNTER_STYLE_RULE = 11;
     [ImplementedAs=Supports] const unsigned short SUPPORTS_RULE = 12;
     [ImplementedAs=FontFeatureValues] const unsigned short FONT_FEATURE_VALUES_RULE = 14;
+    [ImplementedAs=ViewTransition, EnabledBySetting=CSSCrossDocumentViewTransitionsEnabled] const unsigned short VIEW_TRANSITION_RULE = 15;
```

Comment:
I removed this from the spec: https://github.com/w3c/csswg-drafts/commit/f1d42e606113d7a7e7f45fcbd9dad1bd6ce9e775